pom.xml

```diff
@@ -114,7 +114,7 @@
         <dependency>
             <groupId>io.confluent</groupId>
             <artifactId>kafka-connect-storage-hive</artifactId>
-            <version>${kafka.connect.storage.common.version}</version>
+            <version>10.2.0-20210704.081701-800</version>
```

Comment:
any reasons to hardcode here instead of changing the value in the properties section?

Reply:
This is temporary until we can cut a release.
For some reason the -SNAPSHOT release is not working in Jenkins so I tried uploading this specific library manually.